doc/ranger.pod

```diff
@@ -883,6 +883,17 @@ Ask for a confirmation when running the "delete" command?  Valid values are
 "always" (default), "never", "multiple". With "multiple", ranger will ask only
 if you delete multiple files at once.
 
+=item date_format [string]
+
+The date format to be display in the GUI. It defaults to "%x" which is locale's
```

Comment:
"displayed", s/GUI/interface, "the locale"
"for all valid values" -> "for valid format strings"